extensions/ql-vscode/src/view/common/Dropdown.tsx

```diff
@@ -4,7 +4,12 @@ import { styled } from "styled-components";
 
 const DISABLED_VALUE = "-";
 
-const StyledDropdown = styled.select<{ disabled?: boolean }>`
+type FontStyle = "normal" | "italic" | "oblique";
+
+const StyledDropdown = styled.select<{
+  disabled?: boolean;
+  fontStyle?: FontStyle;
+}>`
```

Comment:
Instead of moving the styles down into `Dropdown`, could we keep them in `InProgressDropdown`? To do so, we would need to add a `className` property to `Dropdown` and then use something like this in `InProgressDropdown`:

```ts
const StyledDropdown = styled(Dropdown)`
  font-style: italic;
`;
```

This makes the behaviorial props separate from the styling and avoids having to repeat valid values of `font-style`.

Reply:
D'oh I forgot I could do that.